src/common/utils.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _UTILS_H
+#define _UTILS_H
+
+#include "qrcodegen.h"
+
+
+#define ER_URL "HTTP://HELP.PRUSA3D.COM/"
+#define X0 65
+#define Y0 166
+#define MS 2
+#define BORDER 4
+
+#define qrcodegen_VERSION 9
+#define qrcodegen_ECC qrcodegen_Ecc_HIGH
+#define MAX_LEN_4QR 143
+
+#ifdef __cplusplus
+extern bool createQR(char* str, enum qrcodegen_Ecc qrcodegen_ecl=qrcodegen_ECC);
+extern "C" {
+#endif
+
+
+extern char* eofstr(char* str);
+extern void appendCRC(char* str);
+
+extern void get_path_info(char* str, int error_code);
+extern void create_path_info(char* str, int error_code);
+
+extern bool getQR(char* str, uint8_t* pData, enum qrcodegen_Ecc qrcodegen_ecl);
+extern void drawQR(uint8_t* pData);
```

Comment:
`utils.h` is a very generic name. If something like it should exist, it should include common utility functions related to the project as a whole. E.g., if I include `utils.h`, the last thing I expect to happen is finding mysterious macros like `X0` or `BORDER` floating around.

If such macros are to be part of some header file, please prefix them with their namespace (e.g. `GUI_QR_BORDER` instead of `BORDER`).

What does `get_path_info` or `create_path_info` do? By looking at their signature, I have no idea. Please add some documentation.

Also, note the two used naming styles. Please use snake_case only (`append_crc` instead of `appendCRC`).

Reply:
... upraveno ...